### ISSUE TITLE
update-expectations fails with ValueError: Unknown product <wptrunner.products.Product object at 0x1055c23a0>

### ISSUE THREAD

**Issue by gsnedders** [web-platform-tests/wpt#42823]
```
gsnedders@gsnedders-marsha web-platform-tests % ./wpt update-expectations --metadata metadata --manifest MANIFEST.json --product safari ~/report.json
/Volumes/gsnedders/projects/wpt/web-platform-tests/_venv3/lib/python3.9/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 3.3.6'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
 0:00.17 WARNING Falling back to getting metadata update properties from wptrunner browser product file, this will be removed
Traceback (most recent call last):
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/./wpt", line 10, in <module>
    wpt.main()
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/tools/wpt/wpt.py", line 233, in main
    rv = script(*args, **kwargs)
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/tools/wpt/update.py", line 44, in update_expectations
    update_properties = metadata.get_properties(properties_file=kwargs["properties_file"],
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/tools/wptrunner/wptrunner/metadata.py", line 107, in get_properties
    properties, dependents = products.load_product_update(config, product)
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/tools/wptrunner/wptrunner/products.py", line 54, in load_product_update
    module = product_module(config, product)
  File "/Volumes/gsnedders/projects/wpt/web-platform-tests/tools/wptrunner/wptrunner/products.py", line 9, in product_module
    raise ValueError("Unknown product %s" % product)
ValueError: Unknown product <wptrunner.products.Product object at 0x1055c23a0>
```

It seems like we're expecting a string of the product name here, not an object.

**Comment by gsnedders** [comment]
This is a regression from 9cc314e8042e9637b1eeb1836e462b19fe7730d3 (#39711); -> @jgraham.

**Comment by jgraham** [comment]
https://github.com/web-platform-tests/wpt/pull/42839

**Comment by past** [comment]
Fixed by #42839.
